bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/services/UrlProviderImplTest.java

```diff
@@ -62,11 +71,18 @@
     private MockSlingHttpServletRequest request;
     private CloseableHttpClient httpClient;
     private GraphqlClient graphqlClient;
+    private ComponentsConfiguration caConfig;
+
+    @After
+    public void tearDown() {
```

Comment:
Why is this necessary? Each test runs in isolation and gets a new instance of the test class before setup() is called. That means cqConfig will anyway be set to EMPTY.